src/components/RegistrationSuccessPage/RegistrationSuccessPage.jsx

```diff
@@ -0,0 +1,11 @@
+
+    "use client";
+    import   
+    function Registrationsuccesspage () {
+      return <head>Registration success page</head>
+      function buildURL(item)
```

Comment:
Syntaxis is wrong. Please see React documentation how it should look like, Or check your code from `LoginForm` component.

1. return should contain one element/tag
2. tag/element can have a lot of children
3. `function buildURL(item)` shoul be out of return statement
  You can check this example of component:
  
```
  import React from 'react';
export const renderButton = (props) => {
    const handleClick = () => {
        console.log('handleClick() props data from App: ' + props.dataFromApp)

    }
    return (
        <button onClick={handleClick}>Click</button>
    )
}
```